.pipelines/e2e-windows.yaml

```diff
@@ -116,4 +124,30 @@ stages:
       windowsDistro: aks-windows-2022-containerd
       windowsNodeImageVersion: AKSWindows-2022-containerd-test-2023.02.07
       windowsGpuDriverSuffix: -cuda
-      storageAccount: akswinstore2022cuda
\ No newline at end of file
+      storageAccount: akswinstore2022cuda
+      
+- stage: test_23H2
+  dependsOn: []
+  condition: eq('${{ parameters.test23H2 }}', True)
+  jobs:
+  - template: ./templates/e2e-windows-template.yaml
+    parameters:
+      windowsImage: 23H2
+      windowsOSSKU: WindowsAnnual
+      windowsVMSize: Standard_D2_v2
+      windowsDistro: aks-windows-23H2
+      windowsNodeImageVersion: AKSWindows-23H2-test-2023.02.07
+      storageAccount: akswinstore23H2
+
+- stage: test_23H2_gen2
```

Comment:
Not an issue. Maybe we can keep only one test (default gen1 and later change it to gen2) to save the cost

Reply:
Create a task to track this issue.